saq/queue/redis.py

```diff
@@ -392,12 +392,7 @@ async def finish(
 
             await pipe.execute()
 
-            if status == Status.COMPLETE:
-                self.complete += 1
-            elif status == Status.FAILED:
-                self.failed += 1
-            elif status == Status.ABORTED:
-                self.aborted += 1
+            self._update_stats(status)
 
             await self.notify(job)
```

Comment:
why is conn passed in some places but not others?

Reply:
This is the Redis queue, so there's no conn to pass here. In Postgres, I've centralized most calls to notify() in update() and the remaining call to notify() in finish() passes conn. We generally want conn so notify can be part of conn's transaction.